haskell/cabal.bzl

```diff
@@ -845,6 +900,10 @@ def _compute_dependency_graph(repository_ctx, snapshot, core_packages, versioned
         is_core_package = name in _CORE_PACKAGES
         all_packages[name] = struct(
             name = name,
+            components = _parse_components(
+                name,
+                remaining_components.pop(name, ["lib"]),
```

Comment:
Why only `"lib"`?

Reply:
That forms the default set of package components. I.e. by default we say a package only has a library component. ([`dict.pop(key, default=unbound)`](https://docs.bazel.build/versions/master/skylark/lib/dict.html#pop))